config/router.rb

```diff
@@ -0,0 +1,25 @@
+module Router
+  def specify(route_name)
+    extend(route_module(route_name))
+
+    call
+  end
+
+  class << self
+    def included(_)
+      routes.each(&method(:require))
+    end
+
+    private
+
+    def routes
+      Dir[Rails.root.join('config/routes/**/*.rb')]
+    end
+  end
+
+  private
+
+  def route_module(route_name)
+    "Routes::#{route_name.to_s.camelize}".constantize
```

Comment:
that's not OK, IMO

**We need to:**
1) think how to receive the scope which contains `specify` that has been called;
2) build class name with respective namespacing